test/org/zaproxy/zap/spider/filters/DefaultParseFilterUnitTest.java

```diff
@@ -194,6 +195,54 @@ public void shouldNotFilterHttpMessageWithGitRequest() throws Exception {
         assertThat(filterResult.isFiltered(), is(equalTo(false)));
     }
 
+    @Test
+    public void shouldNotFilterHttpMessageWithRobotsTxtRequestEvenWithoutConentType() throws Exception {
```

Comment:
There's a typo in `content` (same for following test).

Reply:
Good catch, fix inbound.